release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -609,6 +609,11 @@ from both sides of the modal are available for display at one time. Clicking
 *Save* will save the changes that you make. Clicking *Restore Default Columns*
 will restore the default settings of the columns.
 
+[id="ocp-4-6-web-console-insights-plugin"]
+==== Insights plug-in
+
+The Insights plug-in is now integrated into the {product-title} web console. Insights provides cluster health data, such as the number of total issues and total risks of the issues. Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low* and you can quickly navigate to the OpenShift Cluster Manager to learn more about the issues and associated risks.
```

Comment:
`number of total issues` is a bit weird to me

Reply:
@tisnik Can you please clarify what you mean? Is there better phrasing we can use? So that we can get this into GA, I will move forward with getting a peer review and merge it, but please let me know if you have any suggestions that I can apply here or in a follow-up PR. Thanks so much!

Reply:
just "number of total issues" sounds weird, what "total issues" means?